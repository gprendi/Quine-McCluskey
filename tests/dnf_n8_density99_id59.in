d 8
11010000
00100001
00010100
01100100
10011010
10010101
11000000
10010110
01011010
10010100
01011011
10100001
10110111
00110101
00001010
11101111
01110101
01010110
10111011
10000101
00100101
11010100
10001100
01010001
10111100
00010111
11001010
10010111
00100011
11111000
11000101
01110010
01100111
00101100
01111000
01000110
00001100
10011101
01010010
10001011
11100000
11111101
11110101
10001110
00101111
11001110
00011111
11011110
10101101
11000011
01010101
00110110
01011000
11100111
01010111
11100101
11011010
00010010
10000110
11000001
00100100
00010011
00000100
01000000
01110111
00111101
00000000
10011000
01001000
01001011
01111101
11101110
11011101
01000011
01001110
00010001
11100110
10000001
01011101
00100110
10000100
10110011
11101001
11101101
11111011
00110111
01101010
11110000
11000100
11111100
11110011
01101101
11110100
01010011
00001110
00101010
11110010
00110100
10000011
10011110
01111111
10101000
10111010
10010011
00010000
10001010
01110011
00110010
00000001
10111000
01111001
11011011
01101001
11001001
11010101
10111101
01111011
11111111
01011110
01110100
00010101
11111110
01111100
11001111
10101110
11100100
10101111
10100111
00111111
00100010
00101001
10110100
00111011
00001101
11101100
00011001
10001000
00111110
00011110
00111100
01011001
00101101
01010000
10100100
10111111
10100011
10110110
00011010
10001001
01000001
10010001
01000010
10111001
01110001
11101011
11000010
10001111
01001010
10110001
00000010
01100101
01101110
10010000
01101111
00111010
11011100
00011100
11010110
10011100
01110000
01001101
11110110
11111010
10000010
10100110
10000111
10110010
00111000
11100011
01100011
11001000
10000000
00110000
01001100
11011000
01110110
10101100
00011101
11010001
00000111
00000011
11010011
11000111
01111010
01011111
11011111
00110011
00101110
10100010
10110000
11110001
00001111
10011111
11101000
01001111
00111001
11111001
01100010
10101011
11010010
00100111
11101010
01000101
00100000
01100001
10101001
01000111
11000110
00010110
00001001
00001011
01100110
10010010
10101010
01000100
10110101
11001101
01011100
01101100
01111110
10011001
00110001
11100001
11010111
11110111
00000101
10001101
11011001
11100010
10111110
00101000
10011011
00011011
10100101
11001100
01101011
01001001
01100000
01010100
01101000
11001011
00101011
10100000
