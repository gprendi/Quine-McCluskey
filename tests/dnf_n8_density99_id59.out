p dnf 8 0
1 0
2 0
3 0
4 -5 0
4 6 0
4 7 0
-5 -6 0
-5 -7 0
5 6 0
5 7 0
-6 7 0
6 -7 0
8 0
